### ISSUE TITLE
Won't run using PHP 7.0

### ISSUE THREAD

**Issue by f8ttyc8t** [OpenSIPS/opensips-cp#41]
Because I needed to install a new Debian distro (Debian 9 Stretch), PHP 5.x is no longer part of my system.
Now I entirely fail to use control panel.
It seems to fail in MDB2#mysql part - causing error:
PHP Parse error:  syntax error, unexpected 'new' (T_NEW) in /usr/share/php/MDB2.php on line 390

Have tried it with both versions (Control Panel 7.2.3 and 6.2 as well)
Any idea about how to run opensips control panel on a recent Linux distribution?

Thanks in advance!

**Comment by bogdan-iancu** [comment]
@f8ttyc8t , yeah the Control Panel is not php 7 compatible mainly due the deprecated MDB2 . We plan to release a new version of CP for OpenSIPS 2.3, the CP 8.2.3 that will move to PDO for DB ops.

**Comment by f8ttyc8t** [comment]
@bogdan-iancu , thank you very much for letting me know!
Looking forward to new version.

**Comment by bogdan-iancu** [comment]
indeed, there is a bit of a cleanup needed :D, thanks for pointing it out

**Comment by artkos** [comment]
[smonitor-diff.txt](https://github.com/OpenSIPS/opensips-cp/files/2269953/smonitor-diff.txt)

diff file to fix cronjob script :)

**Comment by artkos** [comment]
And to eliminate cron to spam you with:

PHP Warning:  Creating default object from empty value in /var/www/opensips-cp/config/db.inc.php on line 23

You could add following line to db.inc.php:
if (!isset($config)) $config = new stdClass();

**Comment by bogdan-iancu** [comment]
Good job @artkos , thank you - do you want to pack your changes as a pull request, so you will be author of the changes ?

**Comment by bogdan-iancu** [comment]
OK, all the left overs from MDB2 were addressed, so the code should be clean and totally migrated to PDO -> compatible with PHP 7

**Comment by bogdan-iancu** [comment]
@devorel , what version of OpenSIPS Control Panel are you using ?

**Comment by devorel** [comment]
The last.
Mdb2 not running

בתאריך יום ה׳, 16 באוג׳ 2018, 19:05, מאת Bogdan Andrei IANCU ‏<
notifications@github.com>:

**Comment by bogdan-iancu** [comment]
You mean the master git branch ?

**Comment by devorel** [comment]
From the website .


בתאריך יום ה׳, 16 באוג׳ 2018, 19:08, מאת Bogdan Andrei IANCU ‏<
notifications@github.com>:
